compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUVectorLowering.cpp

```diff
@@ -183,6 +184,52 @@ struct SetMulAddFMF final : OpRewritePattern<vector::MultiDimReductionOp> {
   }
 };
 
+struct UnrollElementwiseOps : public RewritePattern {
```

Comment:
```suggestion
struct UnrollElementwise : public RewritePattern {
```

Reply:
I don't know about this, Ops sounds find to me really.